<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Update</class>
 <widget class="QDialog" name="Update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update OpenVR</string>
  </property>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Update in progress. Please wait ...</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="updateProgress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>Update</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
